pages/search/views/index.jsx

```diff
@@ -9,6 +9,16 @@ const formatters = {
       format: (name, establishment) => {
         return <Link page="establishment.dashboard" establishmentId={establishment.id} label={name} />;
       }
+    },
+    inspector: {
+      format: (inspector, establishment) => {
+        return establishment.asru.filter(p => p.asruInspector).map(({ firstName, lastName }) => (firstName + ' ' + lastName));
+      }
+    },
+    spoc: {
+      format: (spoc, establishment) => {
+        return establishment.asru.filter(p => p.asruLicensing).map(({ firstName, lastName }) => (firstName + ' ' + lastName));
```

Comment:
_Very minor_ point of style, and not worth not merging this for, but I'd normally have a preference for template literals over string concatenation.

Reply:
I will fix it with the next ticket, you are absolutely right.